src/Codeception/Module/Doctrine2.php

```diff
@@ -908,7 +910,7 @@ public function grabFromRepository(string $entity, string $field, array $params
      * $users = $I->grabEntitiesFromRepository(User::class, ['name' => 'davert']);
      * ```
      *
-     * @template T
+     * @template T of object
```

Comment:
This fixes the following PHPStan error:

> Unable to resolve the template type T in call to method Doctrine\ORM\EntityManagerInterface::getRepository()

Appears to be the idiomatic solution, especially since it's a [common](https://github.com/doctrine/orm/blob/2.12.x/lib/Doctrine/ORM/EntityManagerInterface.php#L34) [annotation](https://github.com/doctrine/orm/blob/2.12.x/lib/Doctrine/ORM/EntityManager.php#L781) in the Doctrine ORM code itself.